data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,207 @@ mission "Arachi Cartoonist 2"
+				`	"Is it difficult having to deal with all the other houses in the Council?"`
+					goto council
+				`	"This Council is hosted by a royal family, right? What can you tell me about them?"`
+			`	"The ninth house? Not much to say, there is. Agreed as the 'royals' they were, in ancient times. Never much power, they had, merely a figure of stability, made to assure peace on Ahr. Host the Council, they do, but a tradition it is, otherwise mostly to themselves, they keep."`
+				goto end
+			label house
+			`	He grunts lightly. "The deepest mines and biggest smelters in all the Coalition, we have. Overseeing construction, assuring ore purity, testing the quality of alloys. In order, everything must be, so that met, the quotas for house Mallob's and the Heliarchs' ships are."`
+				goto end
+			label council
+			`	"Started out as noble families form Ahr, most of the houses did. When first mapped out, Arach space was, carved their own portions, the houses did, and naturally for more systems, many fought. Now at ease with each other, the houses are, for the most part, and ceased, the territorial disputes have." He sighs. "For the most part."`
+			label end
+			`	Dubug makes it to your ship still awake, but doesn't care to answer when you ask if he needs help unpacking his luggage. Instead he lays on the bed and asks that you wake him up once you arrive on <planet>.`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Dubug hasn't entered the system yet. Better depart and wait for it.`
+	on complete
+		conversation
+			`Having rested during the trip here, Dubug stretches his legs once he steps off your ship. "All the reports from my house, I've brought, so prepared for the Council, I am," he says, checking his suitcase one more time. "Well then, go I must. Goodbye, Captain <last>, thankful for the calm trip, I am. A good break from routine, it was."`
+
+mission "Coalition: Council of Ahr: Bebliss"
+	invisible
+	landing
+	passengers 1
+	blocked "You have reached <origin>, but you need <capacity> in order to take on the next mission. Return here when you have the required space free."
+	to offer
+		has "Coalition: Council of Ahr: active"
+	source "Stronghold of Flugbu"
+	destination "Ahr"
+	on offer
+		conversation
+			`Once your ship has landed, you receive a message from House Bebliss, directing you to a large hotel in the spaceport city. The security guards don't let you in, so you wait for the representative to come out. About half an hour of waiting later, an elevator opens for many Arachi to come out, most of them in formal uniforms and carrying various bags of luggage. There's also an older, and even more formally dressed Arach.`
+			`	"My transport to the Council you are, correct?" She asks, and you confirm. "Splendid. Lady Plubat, I am, Captain. Pleased that transport the will of House Bebliss, such an exquisite captain will. Finally gotten it right, those arranging the transport to the Council have."`
```

Comment:
This sentence jumped out of the page and slapped me.
```suggestion
			`	"My transport to the Council you are, correct?" She asks, and you confirm. "Splendid. Lady Plubat, I am, Captain. Pleased that transport the will of House Bebliss, such an exquisite captain will. Finally right, those arranging the transport to the Council have gotten it."`
```

Reply:
I applied the change manually.